### PR TITLE
Update materials: added extra links in task js basics

### PR DESCRIPTION
**Pull request by boxdxnx**

### CODE REVIEW

File: tasks/js-basics.md [hunk 1]
```diff
@@ -55,8 +55,10 @@ When you finish this task you can proceed to the next one.
 ## Extra materials
 
 - [Курс "Введение в программирование" от Hexlet](https://ru.hexlet.io/courses/introduction_to_programming)
+- [FunFunFunction - Functional programming in JavaScript](https://www.youtube.com/playlist?list=PL0zVEGEvSaeEd9hlmCXrk5yUyqUag-n84)
```
Comment (Croftyland) [inline tasks/js-basics.md:58]: Do we really need those links?

File: tasks/js-basics.md [hunk 2]
```diff
@@ -55,8 +55,10 @@ When you finish this task you can proceed to the next one.
...
 - [Algorithms Course by Khan Academy](https://www.khanacademy.org/computing/computer-science/algorithms)
 - [Visualization of Common Algorithms and Data Structures](https://www.cs.usfca.edu/~galles/visualization/Algorithms.html)
+- [JavaScript Algorithms and Data Structures examples](https://github.com/trekhleb/javascript-algorithms)
```
Comment (Croftyland) [inline tasks/js-basics.md:61]: I think for newbies will be quite complicated and confused for understanding

**Review by AMashoshyna — APPROVED** [review]

File: tasks/js-basics.md (hunk 1 above)
Comment (babayK0) [reply tasks/js-basics.md:58]: FunFunFunction - has a very good explanation of the general principles of functional programming, so i think it was useful for newcomers

File: tasks/js-basics.md (hunk 2 above)
Comment (babayK0) [reply tasks/js-basics.md:61]: This is extra material and optional for further reading. It contains many examples, and someone will be interested to see how to write common algorithms with java script, I guess.

**Review by Croftyland — APPROVED** [review]

**Pull request merged**